sql/mob_pools.sql

```diff
@@ -3004,7 +3004,7 @@ INSERT INTO `mob_pools` VALUES (2943,'Odins_Jester','Odins_Jester',165,0x0000BF0
 INSERT INTO `mob_pools` VALUES (2944,'Odontotyrannus','Odontotyrannus',197,0x00005C0100000000000000000000000000000000,1,1,7,220,100,0,1,0,0,6,0,0,0,3,0,0,0,1,0,197);
 INSERT INTO `mob_pools` VALUES (2945,'Odqan','Odqan',68,0x0000220100000000000000000000000000000000,1,1,5,240,100,0,1,0,0,2,0,32,456,1181,0,0,0,1,0,68);
 INSERT INTO `mob_pools` VALUES (2946,'Odzmanouk','Odzmanouk',260,0x00000F0300000000000000000000000000000000,1,1,0,240,100,0,0,0,0,0,0,0,0,3,0,0,0,1,0,260);
-INSERT INTO `mob_pools` VALUES (2947,'Offspring','Offspring',137,0x0000740400000000000000000000000000000000,1,1,3,240,100,0,1,1,0,16,0,0,5037,1153,0,0,0,1,0,137);
+INSERT INTO `mob_pools` VALUES (2947,'Offspring','Offspring',137,0x0000740400000000000000000000000000000000,1,1,3,240,100,0,1,1,1,16,1,0,5037,1153,0,0,0,1,0,137);
```

Comment:
So I know you haven't touched this skill list ID, but looking at it I see Offspring have the same skill list as Gorgers - is this what it's supposed to have?

Reply:
Yes. Offspring have the same TP abilities as normal Gorgers.